code/entities/weapons/fnb/GooglyEye.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Sandbox;
+
+namespace Cinema;
+
+public partial class GooglyEye : WeaponBase
+{
+    static private float MaxPlacementDistance = 96.0f;
+    PreviewEntity PreviewModel { get; set; }
+
+    public override void ClientSpawn()
+    {
+        base.ClientSpawn();
+
+        if (!PreviewModel.IsValid()) 
+        {
+            PreviewModel = new PreviewEntity
+            {
+                Model = Model.Load("models/googly_eyes/preview_googly_eyes_01.vmdl")
+            };
+        }
+    }
+    protected virtual bool IsPreviewTraceValid(TraceResult tr)
+    {
+        if (!tr.Hit || !tr.Entity.IsValid()) return false;
+
+        if (tr.Entity is GooglyEyeEntity) return false;
+
+        return true;
+    }
+    public override void Simulate(IClient cl)
+    {
+        base.Simulate(cl);
+       
+        if(Game.IsServer)
```

Comment:
Make these into early returns, i'd also get the one on line 38 moved up aswell :)

Reply:
Updated Game.IsServer to be an early return. For line 38.. I'm not sure whether to move it up or not, in case there is something else that needs to be simulated